build.xml

```diff
@@ -306,8 +306,15 @@
         <equals arg1="${ant.java.version}" arg2="17"/>
     </condition>
 
-    <!-- needed to compile org.apache.cassandra.utils.JMXServerUtils -->
-    <condition property="jdk11plus-javac-exports" value="--add-exports java.rmi/sun.rmi.registry=ALL-UNNAMED" else="">
+    <resources id="_jdk11plus_javac_export_args">
+      <!-- needed to compile org.apache.cassandra.utils.JMXServerUtils -->
+      <string>--add-exports java.rmi/sun.rmi.registry=ALL-UNNAMED</string>
+      <!-- needed to compile org.apache.cassandra.utils.memory.BufferPool -->
+      <string>--add-exports java.base/jdk.internal.ref=ALL-UNNAMED</string>
+      <string>--add-exports java.base/sun.nio.ch=ALL-UNNAMED</string>
```

Comment:
@djatnieks We need those also added for IntellijIdea, like this - https://github.com/ekaterinadimitrova2/cassandra/commit/d5fa5ecca4b1fdab27e0002d31bbc581ab5523a8
Please pull this commit. CC @adelapena, as he will also be reviewing, and he will need this one for the IDE. 

In other news, I am still reviewing; I just wanted to ask for this one to be added for everyone's convenience :)

Reply:
Oh, wrong PR/branch :( I will post now on the right one too...Got confused for a moment